app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,8 @@
+<resources>
```

Comment:
Ideally we use only lower case for string resources and then adjust them on use.